packages/vue/src/components/atoms/SfRating/SfRating.vue

```diff
@@ -6,25 +6,61 @@
         :key="`p${index}`"
         size="0.875rem"
         class="sf-rating__icon"
-        icon="star"
+        :icon="icon"
       />
     </slot>
+    <!--@slot custom icon for fractional finalScore. -->
+    <slot name="icon-fraction">
+      <SfIcon v-if="score % 1 !== 0" class="sf-rating__icon" size="0.875rem">
+        <svg
+          class="sf-icon-path"
+          :viewBox="iconViewBox"
+          preserveAspectRatio="none"
+        >
+          <defs>
+            <linearGradient
+              :id="finalScore - Math.floor(finalScore)"
+              x1="0"
+              y1="0"
+              x2="1"
+              y2="0"
+            >
+              <stop
+                :offset="finalScore - Math.floor(finalScore)"
+                stop-color="var(--icon-color)"
+              />
+              <stop offset="0" stop-color="var(--icon-color-negative)" />
+            </linearGradient>
+          </defs>
+          <path
+            v-for="(path, index) in iconPaths"
+            :key="index"
+            :d="path"
+            :fill="fillUrl(finalScore - Math.floor(finalScore))"
+            style="height: 100%;"
+          />
+        </svg>
+      </SfIcon>
+    </slot>
```

Comment:
```suggestion
```
what do you think about use v-for for all positive icons and set right `coverage` for last with fractional?

Reply:
moved to SfIcon